app/assets/javascripts/google_maps.js.es6

```diff
@@ -0,0 +1,24 @@
+function parseMetaTag(name) {
+  const content = $(`meta[name="${name}"]`).attr('content');
+
+  return JSON.parse(content);
+}
+
+function initMap() {
+  const center = parseMetaTag('google-map-center');
+  const donations = parseMetaTag('google-map-markers');
+  const mapOptions = {
+    center: center,
+    mapTypeId: google.maps.MapTypeId.MAP,
+    zoom: 8,
+  };
+  const map = new google.maps.Map(document.getElementById('map'), mapOptions);
+
+  const markers = donations.map(donation => {
+    return new google.maps.Marker({
+      position: donation,
```

Comment:
the constructor allows you to pass `map` in as one of the options, allowing for `forEach` instead of `map` and not having to iterate a second time calling `marker.setMap`